Tests failling:
```
======================================================================
ERROR: corehq.apps.api.odata.tests.test_serializers:TestODataCaseSerializer.test_case_id
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/api/odata/tests/test_serializers.py", line 169, in test_case_id
    0
TypeError: serialize_documents_using_config() missing 1 required positional argument: 'table_id'
```

--- a/corehq/apps/api/odata/serializers.py
+++ b/corehq/apps/api/odata/serializers.py
@@ -71,6 +71,9 @@ class ODataBaseSerializer(Serializer):
         if next_page:
             return '{}{}{}'.format(get_url_base(), api_path, next_page)
 
+    def get_full_row_number(self, row_number):
+        return row_number + (self.offset * self.limit)
+
     def serialize_documents_using_config(self, documents, config, table_id):
         if table_id + 1 > len(config.tables):
             return []
@@ -83,7 +86,7 @@ class ODataBaseSerializer(Serializer):
         for row_number, document in enumerate(documents):
             rows = table.get_rows(
                 document,
-                row_number + (self.offset * self.limit),
+                self.get_full_row_number(row_number),
                 split_columns=config.split_multiselects,
                 transform_dates=config.transform_dates,
                 as_json=True,

--- a/corehq/apps/api/odata/tests/test_serializers.py
+++ b/corehq/apps/api/odata/tests/test_serializers.py
@@ -18,7 +18,7 @@ class TestODataCaseSerializer(SimpleTestCase):
 
     def test_selected_column_included(self):
         self.assertEqual(
-            ODataCaseSerializer.serialize_documents_using_config(
+            ODataCaseSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',
@@ -50,7 +50,7 @@ class TestODataCaseSerializer(SimpleTestCase):
 
     def test_unselected_column_excluded(self):
         self.assertEqual(
-            ODataCaseSerializer.serialize_documents_using_config(
+            ODataCaseSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',
@@ -82,7 +82,7 @@ class TestODataCaseSerializer(SimpleTestCase):
 
     def test_missing_value_is_null(self):
         self.assertEqual(
-            ODataCaseSerializer.serialize_documents_using_config(
+            ODataCaseSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',
@@ -112,7 +112,7 @@ class TestODataCaseSerializer(SimpleTestCase):
 
     def test_non_standard_case_property(self):
         self.assertEqual(
-            ODataCaseSerializer.serialize_documents_using_config(
+            ODataCaseSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',
@@ -143,7 +143,7 @@ class TestODataCaseSerializer(SimpleTestCase):
 
     def test_case_id(self):
         self.assertEqual(
-            ODataCaseSerializer.serialize_documents_using_config(
+            ODataCaseSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': 'case-id-value',
@@ -173,7 +173,7 @@ class TestODataCaseSerializer(SimpleTestCase):
 
     def test_case_name(self):
         self.assertEqual(
-            ODataCaseSerializer.serialize_documents_using_config(
+            ODataCaseSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',
@@ -223,7 +223,7 @@ class TestODataFormSerializer(SimpleTestCase):
 
     def test_selected_column_included(self):
         self.assertEqual(
-            ODataFormSerializer.serialize_documents_using_config(
+            ODataFormSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',
@@ -254,7 +254,7 @@ class TestODataFormSerializer(SimpleTestCase):
 
     def test_unselected_column_excluded(self):
         self.assertEqual(
-            ODataFormSerializer.serialize_documents_using_config(
+            ODataFormSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',
@@ -285,7 +285,7 @@ class TestODataFormSerializer(SimpleTestCase):
 
     def test_missing_value_is_null(self):
         self.assertEqual(
-            ODataFormSerializer.serialize_documents_using_config(
+            ODataFormSerializer().serialize_documents_using_config(
                 [{
                     'domain': 'test_domain',
                     '_id': '54352-25234',

--- a/corehq/apps/export/templates/export/partials/odata_feed_limit_reached_modal.html
+++ b/corehq/apps/export/templates/export/partials/odata_feed_limit_reached_modal.html
@@ -21,7 +21,7 @@
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-default" data-dismiss="modal">
-          Ok
+          {% trans "Ok" %}
         </button>
       </div>
     </div>

